jellyfin.ruleset

```diff
@@ -3,11 +3,19 @@
   <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
     <!-- disable warning SA1101: Prefix local calls with 'this.' -->
     <Rule Id="SA1101" Action="None" />
+    <!-- disable warning SA1130: Use lambda syntax -->
+    <Rule Id="SA1130" Action="None" />
```

Comment:
This rule seems okay to me actually, but I like lambda's a lot.

Reply:
It’s actually for the delegate you commented on earlier.